pandas/tests/indexing/test_loc.py

```diff
@@ -2000,6 +2017,14 @@ def test_loc_setitem_dt64tz_values(self):
         result = s2["a"]
         assert result == expected
 
+    @pytest.mark.parametrize("dtype", ["int64", "Int64"])
+    def test_setitem_series_null_slice_different_dtypes(self, dtype):
```

Comment:
test_loc_setitem_...

Reply:
Thx, changed all test names and fixed failing tests.

Reply:
this one still shows as test_setitem instead of test_loc_setitem

Reply:
Weird, was certain to have changed this. Now it shows correctly